docs/using-jda/interactions.md

```diff
@@ -466,3 +464,113 @@ When a user selects their options from a dropdown and submits their choices, you
             }
         }
         ```
+
+## Modals
+
+Modals are pop-ups that appear in a user's Discord-Client.
+
+![Example Modal](https://i.imgur.com/5r0fEWO.png)
+
+Similarly to messages, Modals can contain up to **5** ActionRows, although the only component that can be put inside Modals at the moment (`TextInput`) takes up a whole ActionRow. 
+
+### Replying with a Modal
+
+!!! example
+    === "Java"
+        ```java
+        public class SupportCommand extends ListenerAdapter {
+            @Override
+            public void onSlashCommandInteraction(SlashCommandInteractionEvent event) {
+                if (event.getName().equals("support")) {
+                    TextInput email = TextInput.create("email", "Email", TextInputStyle.SHORT)
```

Comment:
Why not use the same example from the javadocs, which doesn't have email?

https://ci.dv8tion.net/job/JDA5/javadoc/net/dv8tion/jda/api/interactions/components/Modal.html

Reply:
That's the one i used in the PR description, if you don't like it i can use something else